mkdocs.yml

```diff
@@ -7,7 +7,8 @@ plugins:
   # - mkdocstrings
   - include-markdown
   - git-authors
-  - git-revision-date-localized
+  - git-revision-date-localized:
+      fallback_to_build_date: true
```

Comment:
"Default is false. Enables falling back to the time when mkdocs build was executed when git is not available. This means the revision date will be incorrect, but this can be acceptable if you want your project to also successfully build in environments with no access to GIT."

Did you have no git?

Reply:
I did this because `mkdocs serve` gives this:
```
ERROR   -  [git-revision-date-localized-plugin] Unable to read git logs of '/home/wesley/langdoc-script-collection/docs'.  To ignore this error, set
           option 'fallback_to_build_date: true'
```
Which seems to be because of [this](https://github.com/mkdocs/mkdocs/discussions/2843). I don't think the issue happened due to permissions (all the directories involved are owned by me) so I used this workaround.

I can remove this from the PR and just keep it locally so I can view my docs if that works better.

Reply:
Removed from the PR and I'll just do it locally

Reply:
works fine by me, as long as the live version has correct dates